src/core/lib/event_engine/iomgr_engine/ev_epoll1_linux.cc

```diff
@@ -206,6 +215,14 @@ void ForkPollerListRemovePoller(Epoll1Poller* poller) {
   }
 }
 
+grpc_core::Timestamp ToTimestamp(EventEngine::Duration when) {
```

Comment:
I believe this can be deleted now. And if it's necessary elsewhere, I suggest moving it to the time_util.

Reply:
Deleted this. I didn't need to use it elsewhere so its not added to time_util